.github/workflows/tf-test.yml

```diff
@@ -0,0 +1,64 @@
+name: "TF template tests"
+on: [workflow_dispatch, pull_request]
+
+jobs:
+  ValidateTF:
+    runs-on: ubuntu-latest
+    name: "Validate Terraform module"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: terraform validate
+        uses: dflook/terraform-validate@v1
+        with:
+          directory: 'aws-observability-terraform/'
```

Comment:
rename directory to path `aws-observability-terraform/` since it's giving warning no input variable

Reply:
sure